RELEASING.md

```diff
@@ -1,91 +1,110 @@
+
+2. Checkout a new branch for the release `release-YYYY-MM-DD` (i.e. `release-2024-09-16`)
+
     ```shell
     git checkout -b release-honeycomb-opentelemetry-web-vX.Y.Z
     ```
 
-#### Bump versions and update changelog:
+#### Bump versions and update changelog
+  >
   >For _EACH_ package that has changes to release:
 
 3. `cd` into the package you'd like to release i.e.
+
     ```shell
     cd packages/honeycomb-opentelemetry-web`
     ```
-    OR
-    ```shell
-    cd packages/experimental-opentelemetry-web`
-    ```
+
 4. Clear `node_modules` and install dependencies
+
     ```shell
     rm -rf node_modules
     ```
+
     ```shell
     npm ci
     ```
-5.  Ensure the package builds
+
+5. Ensure the package builds
+
     ```shell
     npm run build
     ```
+
 6. Bump the package version number using `major`, `minor`, `patch`, or the prerelease variants `premajor`, `preminor`, or `prepatch`.
+
    ```shell
    npm version --no-git-tag-version RELEASE_TYPE
    vX.Y.Y
    ```
+
 7. Confirm the version number in the package's `package.json` and `package-lock.json`. The output of `npm version` should match the `version` field in both `package.json` and `package-lock.json`.
 8. Update `version.ts` with the new version number.
 9. Update the `CHANGELOG.md` using:
+
     ```shell
       `git log --pretty='%C(green)%d%Creset- %s | %an'`
     ```
+
     - Copy the latest commits and paste them into the package's `CHANGELOG.md`
     - Update the author names to GitHub handles
 10. If any of the upstream [OpenTelemetry](https://github.com/open-telemetry) package versions have changed, update [`README.md`](./README.md) with new versions and links.
 
 #### Open the release PR
-11. Commit your changes, push a release and open a PR on GitHub (i.e. `git commit -am "rel: Releasing honeycomb-opentelemetry-web-v0.7.0, experimental-opentelemetry-web-v0.1.0"`)
+
+1.  Commit your changes, push a release and open a PR on GitHub (i.e. `git commit -am "rel: Releasing honeycomb-opentelemetry-web-v0.7.0"`)
+
       ```shell
       git commit -am "rel: Releasing TAG_A, TAG_B"
       git push origin BRANCH
       ```
+
       - Make sure the PR title starts with `rel:` to conform with our semantic commits convention
       - Add the `no-changelog`, `type: maintenance`, and `version: no bump` labels
-12. Once the pull request is approved, squash-merge it
-13. Checkout `main` and fetch the now-updated `main` branch.
+2.  Once the pull request is approved, squash-merge it
+3.  Checkout `main` and fetch the now-updated `main` branch.
 
 #### Create tags and publish release
+  >
   >For _EACH_ package that needs to be released:
+
 14. Create a tag for the new version:
+
     ```shell
       git tag -a honeycomb-opentelemetry-web-vX.Y.Z -m honeycomb-opentelemetry-web-vX.Y.Z
     ```
+
     - These tags MUST correspond to the accepted tag prefixes (see [Available Tag Prefixes](#available-tag-prefixes)).
```

Comment:
are we keeping the github tags the same?

Reply:
We could bo back to just using the version number, but I don't have a strong feeling here. So, I'm inclined to leave it as is. Open to contrary opinions!